app/background-process/open-url.js

```diff
@@ -0,0 +1,21 @@
+// handle OSX open-url event
+import { ipcMain } from 'electron'
+var queue = []
+var commandReceiver
+
+export function setup () {
+  ipcMain.on('shell-window-ready', function (e) {
+    commandReceiver = e.sender
+    queue.forEach((url) => {
+      commandReceiver.send('command', 'file:new-tab', url)
+    })
```

Comment:
Need to clear the queue here, otherwise (I think) new windows will open the URLs as well